package/MDAnalysis/core/universe.py

```diff
@@ -850,6 +858,15 @@ def add_TopologyAttr(self, topologyattr, values=None):
                     n_residues=self._topology.n_residues,
                     n_segments=self._topology.n_segments,
                     values=values)
+        alias_pairs = [("bfactors", "tempfactors"),
+                       ("tempfactors", "bfactors")]
+        for a, b in alias_pairs:
+            if topologyattr.attrname == a and hasattr(self._topology, b):
+                warnings.warn(f"You are adding {a} to a Universe that "
```

Comment:
Sigh I forgot this was for `master`. f-strings are a no go unfortunately :(

Reply:
Oops!